app/modules/product/card/Card.tsx

```diff
@@ -1,73 +1,20 @@
 import { FC } from 'react'
-import cl from './Card.module.scss'
-import Image from 'next/image'
-import names from 'classnames'
-import { lato } from '@/app/styles/fonts'
-import { Product } from '@/app/modules/product/types/product.type'
-import Link from 'next/link'
+import FeaturedCard from './variants/featured/FeaturedCard'
+import { Props } from './Card.interface'
+import { assertUnreachable } from '@/app/helpers'
+import BiggerCard from './variants/bigger/BiggerCard'
 
-type Props = {
-  product: Product
-}
+const Card: FC<Props> = ({ product, variant }) => {
+  switch (variant) {
+    case 'featured':
+      return <FeaturedCard product={product} />
+
+    case 'bigger':
+      return <BiggerCard product={product} />
 
-const Card: FC<Props> = ({
-  product: { name, pictureSrc, id, colors, price, tags },
-}) => {
-  return (
-    <div className={cl.card}>
-      <Link href={`/products/${id}`}>
-        <Image
-          src={pictureSrc}
-          alt={name}
-          width={270}
-          height={236}
-          className={cl.image}
-        />
-      </Link>
-      <div
-        className={names(
-          cl.description,
-          lato.className,
-          'flex',
-          'column',
-          'align-center'
-        )}
-      >
-        <h3 className={cl.name}>{name}</h3>
-        <ul className="flex">
-          <li className={cl.colorPaletteListItem}>
-            <input
-              className={cl.colorRadio}
-              type="radio"
-              name={name}
-              value="#05E6B7"
-              style={{ backgroundColor: '#05E6B7' }}
-            />
-          </li>
-          <li className={cl.colorPaletteListItem}>
-            <input
-              className={cl.colorRadio}
-              type="radio"
-              name={name}
-              value="#00009D"
-              style={{ backgroundColor: '#00009D' }}
-            />
-          </li>
-          <li className={cl.colorPaletteListItem}>
-            <input
-              className={cl.colorRadio}
-              type="radio"
-              name={name}
-              value="#FFEAC1"
-              style={{ backgroundColor: '#FFEAC1' }}
-            />
-          </li>
-        </ul>
-        <p className={cl.code}>Code - {id}</p>
-        <p className={cl.price}>${price}.00</p>
-      </div>
-    </div>
-  )
+    default:
+      assertUnreachable(variant)
```

Comment:
You probably don't need to use this, insdead you can make in props some default variant of button, like this `const Card: FC<Props> = ({ product, variant  = 'featured'}) => {...}`